[Docs-2.0] Update UsingWasabi-Send-AnonymitySet (#1177)
zkSNACKs/WasabiDoc: docs/using-wasabi/Send.md

The goal is to know the observers who know about your coins and try to reduce their number for each coin.

## Anonymity Set
A typical bitcoin transaction will contain one input and two outputs - one of the outputs is the coin you are transferring and the other output is the change coin that goes back to your wallet.
Your change coin can be linked to this one input.
There is a 1 in 1 chance to find this link and no plausible deniability.
Thus, Wasabi denotes this coin's anonymity set as `1`.

> In a Wasabi [CoinJoin](/using-wasabi/CoinJoin.md), many peers register coins in the input of the transaction, and in the output there are several equal value coins.

Reviewer: ```suggestion
In a Wasabi [coinjoin](/using-wasabi/CoinJoin.md), many peers register coins in the input of the transaction, and in the output there are several equal value coins.
```

> For example, the output may contain 100 coins worth exactly 0.1 bitcoin.
> This means that when looking at one of these CoinJoin outputs, there is a 1 in 100 chance to find the corresponding input: thus an anonymity set of 100.

Reviewer: IMO we can change these values, because they were more suited for WW1.

```suggestion
For example, the output may contain 20 coins worth exactly 0.3 bitcoin.
This means that when looking at one of these coinjoin outputs, there is a 1 in 20 chance to find the corresponding input: thus an anonymity set of 20.
```

> Therefore, the higher the anonymity set, the more your post-CoinJoin coin is delinked from the pre-CoinJoin history.

Reviewer: ```suggestion
Therefore, the higher the anonymity set, the more your post-coinjoin coin is delinked from the pre-coinjoin history.
```

> There are three CoinJoin strategies to choose from in Wasabi Wallet which effect the target anonymity score for your coins. These three strategies are `Minimize Cost`, `Maximize Speed`, and `Maximize Privacy`. Each of these strategies come with different trade-offs. Alternatively, custom CoinJoin settings can be configured.

Reviewer: Usually we write each sentence in a new line. https://github.com/zkSNACKs/WasabiDoc#writing-conventions
And we don't capitalize the letter `j` in `coinjoin`.

```suggestion
There are three coinjoin strategies to choose from in Wasabi Wallet which effect the target anonymity score for your coins.
These three strategies are `Minimize Cost`, `Maximize Speed`, and `Maximize Privacy`.
Each of these strategies come with different trade-offs. Alternatively, custom coinjoin settings can be configured.
```

> ![CoinJoin Strategy](/CoinjoinStrategy.png "CoinJoin Strategy")

Reviewer: ```suggestion
![Coinjoin Strategy](/CoinjoinStrategy.png "Coinjoin Strategy")
```

## Anonymity Set
A typical bitcoin transaction will contain one input and two outputs - one of the outputs is the coin you are sending and the other output is the change coin that goes back to your wallet.
Your change coin can be linked to this one input.
There is a 1 in 1 chance to find this link and no plausible deniability.
Thus, Wasabi denotes this coin's anonymity set as `1`.

In a Wasabi [coinjoin](/using-wasabi/CoinJoin.md), many peers register coins in the input of the transaction, and in the output there are several equal value coins.
For example, the output may contain 20 coins worth exactly 0.3 bitcoin.
This means that when looking at one of these coinjoin outputs, there is a 1 in 20 chance to find the corresponding input: thus an anonymity set of 20.
Therefore, the higher the anonymity set, the more your post-coinjoin coin is delinked from the pre-coinjoin history.

There are three coinjoin strategies to choose from in Wasabi Wallet which effect the target anonymity score for your coins.
These three strategies are `Minimize Cost`, `Maximize Speed`, and `Maximize Privacy`.

> Each of these strategies come with different trade-offs. Alternatively, custom coinjoin settings can be configured.

Reviewer: ```suggestion
Each of these strategies come with different trade-offs.

Alternatively, custom coinjoin settings can be configured.
```

The goal is to know the observers who know about your coins and try to reduce their number for each coin.

## Anonymity Set

> A typical bitcoin transaction will contain one input and two outputs - one of the outputs is the coin you are transferring and the other output is the change coin that goes back to your wallet.

Reviewer: ```suggestion
A typical bitcoin transaction will contain one input and two outputs - one of the outputs is the coin you are sending and the other output is the change coin that goes back to your wallet.
```

Your change coin can be linked to this one input.
There is a 1 in 1 chance to find this link and no plausible deniability.
Thus, Wasabi denotes this coin's anonymity set as `1`.

In a Wasabi [coinjoin](/using-wasabi/CoinJoin.md), many peers register coins in the input of the transaction, and in the output there are several equal value coins.
For example, the output may contain 20 coins worth exactly 0.3 bitcoin.
This means that when looking at one of these coinjoin outputs, there is a 1 in 20 chance to find the corresponding input: thus an anonymity set of 20.
Therefore, the higher the anonymity set, the more your post-coinjoin coin is delinked from the pre-coinjoin history.

There are three coinjoin strategies to choose from in Wasabi Wallet which effect the target anonymity score for your coins.
These three strategies are `Minimize Cost`, `Maximize Speed`, and `Maximize Privacy`.
Each of these strategies come with different trade-offs. Alternatively, custom coinjoin settings can be configured.

![Coinjoin Strategy](/CoinjoinStrategy.png "Coinjoin Strategy")

> 

Reviewer: ```suggestion
```
